system/include/emscripten/vr.h

```diff
@@ -119,8 +119,9 @@ typedef struct VRFrameData {
     VRPose pose;
 } VRFrameData;
 
-extern int emscripten_vr_init(void);
 extern int emscripten_vr_ready(void);
+extern int emscripten_vr_init(em_vr_arg_callback_func callback, void* userData);
+extern int emscripten_vr_deinit();
```

Comment:
This would be good to be `extern int emscripten_vr_deinit(void);`, without the `void` it will be interpreted as `extern int emscripten_vr_deinit(...);` by C standard. That doesn't affect anything in practice, except that it can create a spurious warning message `mismatching number of parameters to input function, saw 0 expected 1`, or similar.